src/test/resources/sql/get-all-premises.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO ap_area (id, name) VALUES
+    ('APAREA1', 'APAREA1'),
+    ('APAREA2', 'APAREA2');
+
+INSERT INTO local_authority_area (id, name) VALUES
+    ('ATAREA1', 'ATAREA1'),
+    ('ATAREA2', 'ATAREA2');
+
+INSERT INTO probation_region (id, name) VALUES
+    ('PBREG1', 'PBREG1'),
+    ('PBREG2', 'PBREG2');
+
+INSERT INTO premises (id, name, ap_code, postcode, probation_region_id, ap_area_id, local_authority_area_id) VALUES
+    ('322e3614-2dc0-4166-8dec-63c016e20a0e', 'Premises One', 'PREMONE', 'ST8ST8', 'PBREG1', 'APAREA1', 'ATAREA1'),
+    ('f04743a7-d9f1-4818-8847-5c15f4874ec0', 'Premises Two', 'PREMTWO', 'ST8ST7', 'PBREG2', 'APAREA2', 'ATAREA2');
+
+INSERT INTO bed (id, premises_id) VALUES
+    ('62a24e1c-7e4a-4dc3-b52e-a1495a3e8937', '322e3614-2dc0-4166-8dec-63c016e20a0e'),
+    ('7984554a-83f9-4651-9d4b-b4898803273b', 'f04743a7-d9f1-4818-8847-5c15f4874ec0'),
+    ('04bc06f3-0efb-448e-a129-5ef858f15cbd', 'f04743a7-d9f1-4818-8847-5c15f4874ec0');
```

Comment:
Is there an easier way to generate objects here, rather than directly using SQL? Thinking about something like https://github.com/thoughtbot/fishery (in Typescript-land), or https://github.com/thoughtbot/factory_bot (in Ruby land). Just there's a lot of boilerplate that needs to be done to setup/teardown

Reply:
I think the closest to those we'll get is using JPA from the tests to save objects - will switch it over so you can take a look.

Reply:
Have switched it over now - more LOC overall so not sure it really adresses your concern around boilerplate?  I can't think of a way to do this any more succintly though unfortunately.